eva_assembly_ingestion/job.py

```diff
@@ -0,0 +1,387 @@
+# Copyright 2022 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import datetime
+import os
+import subprocess
+
+import yaml
+from cached_property import cached_property
+from ebi_eva_common_pyutils.command_utils import run_command_with_output
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.config_utils import get_contig_alias_db_creds_for_profile
+from ebi_eva_common_pyutils.contig_alias.contig_alias import ContigAliasClient
+from ebi_eva_common_pyutils.logger import AppLogger
+from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle, insert_new_assembly_and_taxonomy
+from ebi_eva_common_pyutils.pg_utils import execute_query, get_all_results_for_query
+from ebi_eva_common_pyutils.spring_properties import SpringPropertiesGenerator
+from ebi_eva_common_pyutils.taxonomy.taxonomy import get_scientific_name_from_taxonomy
+from psycopg2.extras import execute_values
+
+from eva_assembly_ingestion.parse_counts import count_variants_extracted, count_variants_remapped, \
+    count_variants_ingested
+
+
+def pretty_print(header, table):
+    cell_widths = [len(h) for h in header]
+    for row in table:
+        for i, cell in enumerate(row):
+            cell_widths[i] = max(cell_widths[i], len(str(cell)))
+    format_string = ' | '.join('{%s:>%s}' % (i, w) for i, w in enumerate(cell_widths))
+    print('| ' + format_string.format(*header) + ' |')
+    for row in table:
+        print('| ' + format_string.format(*row) + ' |')
```

Comment:
Since we are doing a PR in pyutils anyway, may be we can add this in that PR. This could be handy in other contexts as wel...